crates/sui-rpc-api/proto/sui/rpc/v2alpha/event.proto

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package sui.rpc.v2alpha;
+
+import "sui/rpc/v2alpha/bcs.proto";
+
+// Events emitted during the successful execution of a transaction.
+message TransactionEvents {
+  // This TransactionEvents serialized as BCS.
+  optional Bcs bcs = 1;
+
+  // The digest of this TransactionEvents.
+  optional string digest = 2;
+
+  // Set of events emitted by a transaction.
+  repeated Event events = 3;
+}
+
+// An event.
+message Event {
+  // Package ID of the top-level function invoked by a `MoveCall` command that triggered this
+  // event to be emitted.
+  optional string package_id = 1;
+
+  // Module name of the top-level function invoked by a `MoveCall` command that triggered this
+  // event to be emitted.
+  optional string module = 2;
+
+  // Address of the account that sent the transaction where this event was emitted.
+  optional string sender = 3;
+
+  // The type of the event emitted.
+  optional string event_type = 4;
+
+  // BCS serialized bytes of the event.
+  optional bytes contents = 5;
```

Comment:
It's a little confusing that this doesn't use the `Bcs` type -- maybe this is related to the question I had about the `name` field there -- arguably `event_type` is the `name` for this BCS.

Reply:
I'm thinking if we want consistency that we could use `Bcs` for the contents field, have the type be the `name` but also keep `event_type` around. that does lead to the type information possibly being duplicated of course.